content/resources/help-centre/software/unsigned/index.md

```diff
@@ -23,6 +27,30 @@ unlock button, and then tick Terminal:
 Now the terminal will be able to run programs that do not meet the security
 policy. 
 
+## Help! I can't see that setting
+
+If you're using an older version of macOS, such as High Sierra, this option may
+not be visible. That's ok, because we can also disable the protection by using a
+command in our terminal. This method will also work on recent versions,
+such as Monterey. 
+
+```sh
+> sudo spctl --master-disable
+```
+
+The `sudo` command lets you run commands with elevated privileges, so you will
+be prompted to input your password. After successfully running the command
+above, you will be able to open unsigned programs using the terminal. If you
+wish to enable the protection again later, you can use:
+
+```sh
+> sudo spctl --master-enable
+```
+
+Note that once you have successfully opened and run an unsigned program after
+disabling the protection, that program will continue to be available, even after
+you re-enable the protection. 
+
```

Comment:
Ok I researched this. This is a more dangerous change: it disabled gatekeeper system wide and not just for terminal apps.

We should add a strong warning (`danger`) that this is not an ideal solution, and `spctl` should be reenabled as soon as we're done allowing the target files to run.

Reply:
Ok  i'll add a warning to this. 

That github link is super helpful. I tested their example to disable the quarantine for a single app using 
`xattr -dr com.apple.quarantine ./emu` 
And this worked for me, letting me run emu while my gatekeeper was enabled. So i'll add that in as well, and the link to that github.